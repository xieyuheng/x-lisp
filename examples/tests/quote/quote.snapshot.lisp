'123
'abc
'"a b c"
'""
'(a)
'(1 2 3)
'(a b c 1 2 3 "a b c")

(@quote 123)
(@quote abc)
(@quote "a b c")
(@quote "")
(@quote (a))
(@quote (1 2 3))
(@quote (a b c 1 2 3 "a b c"))

'(a b c)
'[a b c]
'{a b c}

''a
'`a
'(`(a))
'(`(a ,x b))
