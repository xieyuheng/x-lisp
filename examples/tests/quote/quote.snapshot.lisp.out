123
'abc
"a b c"
""
['a]
[1 2 3]
['a 'b 'c 1 2 3 "a b c"]
123
'abc
"a b c"
""
['a]
[1 2 3]
['a 'b 'c 1 2 3 "a b c"]
['a 'b 'c]
['@tael 'a 'b 'c]
['@set 'a 'b 'c]
['@quote 'a]
['@quasiquote 'a]
[['@quasiquote ['a]]]
[['@quasiquote
  ['a ['@unquote 'x] 'b]]]
